apps/system/locales/system.en-US.properties

```diff
@@ -209,12 +209,12 @@ downloadingAppProgress={{ progress }} / {{ max }} downloaded
 cancelAllDownloads=Cancel all downloads
 wantToCancelAll=Are you sure you want to cancel all downloads?
 downloadError=There was an error while downloading the updates.
-updateAvailableInfo.innerHTML        = {[ plural(n) ]}
-updateAvailableInfo.innerHTML[one]   = {{ n }} update available. <span>Tap for more info.</span>
-updateAvailableInfo.innerHTML[two]   = {{ n }} updates available. <span>Tap for more info.</span>
-updateAvailableInfo.innerHTML[few]   = {{ n }} updates available. <span>Tap for more info.</span>
-updateAvailableInfo.innerHTML[many]  = {{ n }} updates available. <span>Tap for more info.</span>
-updateAvailableInfo.innerHTML[other] = {{ n }} updates available. <span>Tap for more info.</span>
+updateAvailableInfo        = {[ plural(n) ]}
+updateAvailableInfo[one]   = {{ n }} update available. <span>Tap for more info.</span>
```

Comment:
Same here, how is the tap on this `<span>` handled?  Do we need to add it to the source HTML?